src/pyhf_benchmark/manager.py

```diff
@@ -1,4 +1,5 @@
 import time
+import pyhf_benchmark
```

Comment:
Importing a package inside of itself is going to be problematic and will lead to conflicts.
```suggestion
```